#!/usr/bin/env python3
PACKAGE = "es_image_to_steering"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("enable", bool_t, 0, "Enable es_image_to_steering", False)
gen.add("linear_speed", double_t, 0, "Linear speed", 2.0, 0.0, 5.0)
gen.add("steering_gain", double_t, 0, "Steering gain", 1.0, 0.0, 2.0)


exit(gen.generate(PACKAGE, "es_image_to_steering", "EsSteering"))
